content/altinn-studio/guides/development/signing/runtime-delegated-signing/backend-manual/signee-provider.en.md

```diff
@@ -0,0 +1,132 @@
+---
+hidden: true
+---
+
+To allow the app to determine who should receive access to read and sign, the C# interface `ISigneeProvider` must be implemented.
+
+The implementation must return a set of individuals and/or organizations that should receive rights. This can be based on the data model, as shown in the example below.
```

Comment:
```suggestion
The implementation must return a set of individuals and/or organizations that should receive rights to sign. This can be based on the data model, as shown in the example below.
```